timetomodel/speccing.py

```diff
@@ -266,20 +311,8 @@ def resample_data(
             },
```

Comment:
I believe `event_resolution` is also not an expected argument for `data.resample()`?

Reply:
Correct. In the code I pop this argument from the `resampling_config` dictionary, and I've now revised the docstring of the `load_series` method accordingly.